src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -60,6 +60,8 @@
   private static final String HEC_SOURCE_TYPE_KEY = "sourcetype";
 
   private static final String TIMESTAMP_KEY = "timestamp";
+  
+  private static final String DATA = "data";
```

Comment:
Possible to re-use same constant `PUBSUB_MESSAGE_DATA_FIELD`?
https://github.com/melbrodrigues/DataflowTemplates/blob/incompatible-ps-msg/src/main/java/com/google/cloud/teleport/templates/PubSubToSplunk.java#L160

Reply:
yes, this is just a placeholder till the other PR is merged

Reply:
Renamed to `PUBSUB_MESSAGE_DATA_FIELD` per our last discusstion.